### DISCUSSION TITLE
[Documentation] Docker Instructions: Commands Missing Parameters

### DISCUSSION THREAD

**Discussion by pennro-se** [outline/outline#4003] [Self Hosting] [answered]
Hello,

When trying to set up an Outline install today I came across this issue with the [Docker Install Instructions](https://app.getoutline.com/share/770a97da-13e5-401e-9f8a-37949c19f97e/doc/docker-7pfeLP5a8t):

`docker-compose run --rm outline yarn db:create --env=production-ssl-disabled`
and `docker-compose run --rm outline yarn db:migrate --env=production-ssl-disabled` are missing the `sequelize` parameter.

Instead they should be `docker-compose run --rm outline yarn sequelize db:create --env=production-ssl-disabled`
and `docker-compose run --rm outline yarn sequelize db:migrate --env=production-ssl-disabled` as per [this discussion answer](https://github.com/outline/outline/discussions/3280#discussioncomment-2453745).

I can confirm that the lack of `sequelize` as of v0.65.2 does not work.

Thanks

**Comment by tommoor** [comment, marked as answer]
Thanks, `db:create` command is indeed missing without the sequelize

**Reply by pennro-se** [reply]
Are there any plans to update the documentation to fix this?

**Reply by tommoor** [reply]
I plan on releasing a new build in the coming days. The documentation will be correct again then
